component/component/vshn_redis.jsonnet

```diff
@@ -377,6 +396,16 @@ local composition =
             version: redisParams.helmChartVersion,
           },
           values: {
+            metrics: {
+              enabled: true,
+              containerSecurityContext: {
+                enabled: false,
```

Comment:
This I suppose is to make it work on OpenShift?

Reply:
exactly

Reply:
Don't we have a variable to figure out if the cluster has distribution OpenShift? We want our AppCat to run also on on OpenShift clusters. Like [here](https://github.com/vshn/component-appcat/blob/master/component/component/vshn_redis.jsonnet#L37).